### DISCUSSION TITLE
Make SchemaVersion attribute useable in mods

### DISCUSSION THREAD

**Discussion by xmnovotny** [voxeltycoon/issues#784] [Modding] [answered]
SchemaVersionCollection collects SchemaVersion attributes only for game classes, so SchemaVersion<T> cannot be used in mod classes for versioning the saved data. 
It would be useful to make it work for all classes in all assemblies.

**Comment by andrewpey** [comment, marked as answer]
Good point, thanks. Will be implemented with the next update.

**Reply by xmnovotny** [reply]
Great! Thanks.

**Reply by andrewpey** [reply]
The latest build on the
[beta branch](https://voxeltycoon.xyz/devlog/beta-branch/) now has this feature implemented.
